web/src/copy_and_paste.ts

```diff
@@ -154,7 +154,36 @@ function remove_div(_div: JQuery, ranges: Range[]): void {
     }, 0);
 }
 
-export function copy_handler(): void {
+async function copy_selection_to_clipboard(selection: Selection): Promise<void> {
+    const range = selection.getRangeAt(0);
+    const div = document.createElement("div");
+    div.append(range.cloneContents());
```

Comment:
Can you explain why you need to create this `div` here? I'd have expected that the browser `Selection` APIs offered a way to more directly access the HTML in a selection range. E.g., https://stackoverflow.com/questions/19861085/range-selection-get-html#comment29540042_19861085 suggests 
 `range.cloneContents().childNodes[1].innerHTML` without creating a new DOM node.

Reply:
OK. It's not the end of the world to use that strategy; we already do something like it in `construct_copy_div`. But I'm not sure why we'd need to do it twice in a single copy operation.